src/app/shared/components/Field/index.tsx

```diff
@@ -0,0 +1,62 @@
+import Image from 'next/image';
+
+import PlayerCard from './PlayerCard';
+
+const fieldData = [
+  { key: 2, position: 'prop', className: 'col-start-3 col-span-2' },
+  { key: 1, position: 'hooker', className: 'col-span-2' },
+  { key: 3, position: 'prop', className: 'col-span-2'  },
+  { key: 4, position: 'lock', className: 'col-start-4 col-span-2' },
+  { key: 5, position: 'lock', className: 'col-span-2' },
+  { key: 6, position: 'backrow', className: 'col-start-3 col-span-2' },
+  { key: 7, position: 'backrow', className: 'col-span-2' },
+  { key: 8, position: 'backrow', className: 'col-span-2' },
+  { key: 9, position: 'scrumhalf', className: 'col-start-3 col-span-2' },
+  { key: 10, position: 'flyhalf', className: 'col-start-4 col-span-2' },
+  { key: 12, position: 'centre', className: 'col-start-5 col-span-2' },
+  { key: 13, position: 'centre', className: ' col-start-7 col-span-2' },
+  { key: 11, position: 'outsideBack', className: 'col-start-2 col-span-2' },
+  { key: 14, position: 'outsideBack', className: 'col-start-8 col-span-2' },
+  { key: 15, position: 'outsideBack', className: 'col-start-5 col-span-2' },
+]
+
+const Field = () => (
```

Comment:
i notice the field does not go all the way to the edge of the screen, couldn't find a simple fix since it seems to be in the svg itself

if you can't find an easy fix lets just merge this and come back later

![Screenshot 2023-11-28 at 7 42 05 am](https://github.com/dcarg/teamsheet/assets/54693327/11915edc-f1fa-463b-ace8-de7ed945e701)